Block/BreadcrumbBlockService.php

```diff
@@ -32,16 +34,29 @@ class BreadcrumbBlockService extends BaseBreadcrumbMenuBlockService
     protected $cmsSelector;
 
     /**
+     * @var RequestStack
+     */
+    protected $requestStack;
+
+    /**
+     * @var ContainerInterface
+     */
+    protected $container;
```

Comment:
u mean use protected $defaultLocale, and in construct i do something like $defaultLocale = $container->getParameter('locale'); ?